test/RpcCompatibility.js

```diff
@@ -117,11 +125,19 @@ describe('RpcCompatibility', () => {
             await checkLog(filter);
         });
 
-        it('supports finalized tag', async () => {
-            let isLarger = false;
+        describe('supports finalized tag', () => {
+            let finalizedBlockNumber;
 
-            try {
-                const finalizedBlockNumber = await provider.send('eth_getBlockByNumber', ['finalized', false]);
+            before(async () => {
+                try {
+                    finalizedBlockNumber = await provider.send('eth_getBlockByNumber', ['finalized', false]);
+                } catch (error) {
+                    console.error('Failed to retrieve finalized block number:', error);
```

Comment:
```suggestion
                    console.error('Failed to retrieve finalized block number:', error);
                    throw error;
```
throw again to fail, no point in running the below tests if it's not supported